huaweicloud/services/cfw/resource_huaweicloud_cfw_ip_address_group.go

```diff
@@ -0,0 +1,254 @@
+// ---------------------------------------------------------------
+// *** AUTO GENERATED CODE ***
+// @Product CFW
+// ---------------------------------------------------------------
+
+package cfw
+
+import (
+	"context"
+	"strings"
+
+	"github.com/hashicorp/go-multierror"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+	"github.com/jmespath/go-jmespath"
+
+	"github.com/chnsz/golangsdk"
+
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/common"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/config"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/utils"
+)
+
+func ResourceIPAddressGroup() *schema.Resource {
+	return &schema.Resource{
+		CreateContext: resourceIPAddressGroupCreate,
+		UpdateContext: resourceIPAddressGroupUpdate,
+		ReadContext:   resourceIPAddressGroupRead,
+		DeleteContext: resourceIPAddressGroupDelete,
+		Importer: &schema.ResourceImporter{
+			StateContext: schema.ImportStatePassthroughContext,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"region": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Computed: true,
+				ForceNew: true,
+			},
+			"object_id": {
+				Type:        schema.TypeString,
+				Required:    true,
+				ForceNew:    true,
+				Description: `Specifies the protected object ID.`,
+			},
+			"name": {
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: `Specifies the IP address group name.`,
+			},
+			"address_type": {
+				Type:         schema.TypeInt,
+				Optional:     true,
+				Computed:     true,
+				ForceNew:     true,
+				Description:  `Specifies the Address type.`,
+				ValidateFunc: validation.IntInSlice([]int{0, 1}),
+			},
+			"description": {
+				Type:        schema.TypeString,
+				Optional:    true,
+				Description: `Specifies the Address group description.`,
+			},
+		},
```

Comment:
The API request header (address group and member) already supports `enterprise_project_id`, shall we support it?